metaflow/plugins/kfp/tests/run_integration_tests.py

```diff
@@ -51,10 +58,53 @@ def obtain_flow_file_paths(flow_dir_path: str) -> List[str]:
         and not file_name.startswith(".")
         and not "raise_error_flow" in file_name
         and not "accelerator_flow" in file_name
+        and not "s3_sensor_flow" in file_name
+        and not "upload_to_s3_flow" in file_name
```

Comment:
This not in list can be a list, and instead you perform `and not file_name in exclude_list_something`

Reply:
Good idea, this is becoming overwhelming!